src/simulator/halo_simulator.py

```diff
@@ -243,40 +243,54 @@ def simulated_venn_diagram_reach_by_spend(
     def _form_venn_diagram_regions(
         self, spends: List[float], max_frequency: int = 1
     ) -> Dict[int, List]:
-        """Form Venn diagram regions that contain k+ reaches
+        """Form primitive Venn diagram regions that contain k+ reaches
 
-        For each subset of publishers, computes k+ reaches for those users
-        who are reached by the publishers in that subset.
+        For each subset in the powerset of publishers, computes k+ reaches for
```

Comment:
I think it might be better to write, "For each subset in the powerset of publishers with nonzero spend, ..."

Reply:
Yes it is! I will change to the suggested wording.

Reply:
Done.